src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,127 @@
 {% endblock %}
 
 {% block content %}
-<section class="content">
+<div class="content">
 
   <section id="title">
-      <h1 class="title title-center">{{ self.intro_title() }}</h1>
-      <h2 class="title title-center">{{ self.intro_sub_title() }}</h2>
-
-      <img class="intro-image" src="/static/images/home-hero.png" alt="" width="984" height="354" loading="lazy" />
-    </section>
-
-    <section>
-      <h1>{{ self.about_title() }}</h1>
-
-      {{ self.mission() }}
-
-      <div class="contributors">
-        <h2 class="title">{{ self.contributors_title() }}</h2>
-        <p>{{ self.contributors_description() }}
-          <a href="#contributors" class="alt">
-            {{ self.contributors_link() }}
-          </a>
-        </p>
-      </div>
-
-      <div class="methodology">
-        <h2 class="title">{{ self.methodology_title() }}</h2>
-        <p class="methodology-info">
-          {{ self.methodology_description() }}
-          <a href="#methodology" class="alt">
-            {{ self.methodology_link() }}
-          </a>
-        </p>
-        <img class="methodology-characters" src="/static/images/methodology-characters.png" alt="" width="984" height="354" loading="lazy" />
+    <h1 class="title title-center">{{ self.intro_title() }}</h1>
+    <h2 class="title title-center">{{ self.intro_sub_title() }}</h2>
+
+    <img class="intro-image" src="/static/images/home-hero.png" alt="" width="984" height="354" />
+  </section>
+
+  <section>
```

Comment:
I'd suggest removing this page. Feels out of place.

Reply:
OK. Was trying to reuse the nice content from Home Page without dumping this on front cover. will remove then.

Reply:
It's gone.